Microsoft.Toolkit.Uwp/Helpers/HttpHelper/HttpHelper.cs

```diff
@@ -55,13 +61,48 @@ protected HttpHelper()
         /// <returns>Instane of <see cref="HttpHelperResponse"/></returns>
         public async Task<HttpHelperResponse> SendRequestAsync(HttpHelperRequest request)
         {
-            var httpRequestMessage = request.ToHttpRequestMessage();
+            await _semaphore.WaitAsync().ConfigureAwait(false);
+
+            HttpClient client = null;
 
-            var response = await _httpClient.SendRequestAsync(httpRequestMessage).AsTask().ConfigureAwait(false);
+            try
+            {
+                var httpRequestMessage = request.ToHttpRequestMessage();
+
+                client = GetHttpClientInstance();
```

Comment:
This could potentially be null as the GetHttpClientInstance() method allows null to be returned. This would cause the next line of code to throw a nullreferenceexception.